### PR TITLE
JavaScript function to check if a field is empty 

### PR DESCRIPTION
**Pull request by Stanislava88**
In this PR i have solved the task: Create JavaScript function to check if a field is empty and if it is empty a message appears on the screen.

Correct field:
notemptyfield

empty field:
emptyfield

### CODE REVIEW

File: src/com/clouway/field/field.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Created by clouway on 30.05.16.
+ */
+function validateField() {
```
Comment (freekman) [inline src/com/clouway/field/field.js]: rename just to validate

File: src/com/clouway/field/field.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    var x = document.getElementById("field").value;
```
Comment (freekman) [inline src/com/clouway/field/field.js]: ```x``` is a very nice variable name ```The X field```,.............

File: src/com/clouway/field/field.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    var message;
```
Comment (freekman) [inline src/com/clouway/field/field.js]: this can be made as 
```js 
var err = ""

and the else cloause could be removed
````

**Comment by freekman** [comment]
LGTM

File: src/com/clouway/field/field.js (hunk 1 above)
Comment (Stanislava88) [inline src/com/clouway/field/field.js]: done

File: src/com/clouway/field/field.js (hunk 2 above)
Comment (Stanislava88) [inline src/com/clouway/field/field.js]: done

File: src/com/clouway/field/field.js (hunk 3 above)
Comment (Stanislava88) [inline src/com/clouway/field/field.js]: done

**Pull request merged**